documentation/userguide/docs/metadata_forms.md

```diff
@@ -0,0 +1,116 @@
+# **Metadata Forms**
+
+**Introduction**
+
+Metadata forms allow users to add structured contextual information to various entities in the data.all platform. By creating and attaching metadata forms, user can standardize and enrich metadata in a customizable way.
```

Comment:
ultra nit: user -> users

Reply:
or a data.all user

Reply:
This happens a couple of times in the file, we use "user" without any article (a/an/the) which I think might be a small grammar error